client/graphics/shader.hpp

```diff
@@ -83,37 +83,40 @@ namespace tec {
 		 * Filenames should be something like:
 		 * \code{.cpp}:
 		 * auto shader_files = std::list < std::pair<Shader::ShaderType, std::string> > {
-		 *	std::make_pair(Shader::VERTEX, FilePath("basic.vert")), std::make_pair(Shader::FRAGMENT, FilePath("./basic.frag")),
+		 *	std::make_pair(Shader::VERTEX, FilePath("basic.vert")), std::make_pair(Shader::FRAGMENT,
+		 *FilePath("./basic.frag")),
 		 * };
 		 * \return is a weak_ptr to the created Shader.
 		 */
-		static std::shared_ptr<Shader> CreateFromFile(const std::string name, std::list<std::pair<Shader::ShaderType, tec::FilePath>> filenames);
-	
+		static std::shared_ptr<Shader> CreateFromFile(const std::string name,
+			std::list<std::pair<Shader::ShaderType, tec::FilePath>> filenames);
+
 		/**
 		 * \brief Creates a Shader the provide source code stores it in ShaderMap under name.
 		 *
 		 * Source_code should be something like:
 		 * \code{.cpp}:
 		 * auto shader_files = std::list < std::pair<Shader::ShaderType, std::string> > {
-		 *   std::make_pair(Shader::VERTEX, "shader source code\nversion 330\n{}"), std::make_pair(Shader::FRAGMENT, "shader source code\nversion 330\n{}"),
+		 *   std::make_pair(Shader::VERTEX, "shader source code\nversion 330\n{}"),
+		 * std::make_pair(Shader::FRAGMENT, "shader source code\nversion 330\n{}"),
 		 * };
 		 * \return is a weak_ptr to the created Shader.
 		 */
-		static std::shared_ptr<Shader> CreateFromString(const std::string name, std::list<std::pair<Shader::ShaderType, std::string>> source_code);
+		static std::shared_ptr<Shader> CreateFromString(const std::string name,
+			std::list<std::pair<Shader::ShaderType, std::string>> source_code);
 
 		/**
 		 * \brief Loads the specified ShaderType from file filename.
-		 * \param const ShaderType type The type of shader that is being loaded (VERTEX, FRAGMENT, GEOMETRY).
-		 * \param const FilePath filename The filename of the source file to load (relative to assets folder)
-		 * \return void
+		 * \param const ShaderType type The type of shader that is being loaded (VERTEX, FRAGMENT,
+		 * GEOMETRY). \param const FilePath filename The filename of the source file to load
+		 * (relative to assets folder) \return void
 		 */
 		void LoadFromFile(const Shader::ShaderType type, const tec::FilePath& filename);
 
 		/**
 		 * \brief Loads the specified ShaderType from the source string provided..
-		 * \param const ShaderType type The type of shader that is being loaded (VERTEX, FRAGMENT, GEOMETRY).
-		 * \param const std::string source The source string to load from.
-		 * \return void
+		 * \param const ShaderType type The type of shader that is being loaded (VERTEX, FRAGMENT,
```

Comment:
we can't use this if it's going to fuck up all the doc comments

Reply:
This could be due to `ReflowComments` which I can see about disabling and run the pass again.